pages/legacy/workspace_collection.py

```diff
@@ -0,0 +1,37 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from regions.webview.base import Region
+from pages.legacy.workspace import Workspace
+
+from selenium.webdriver.common.by import By
+
+
+class WorkspaceCollection(Workspace):
+    _collection_locator = (By.CSS_SELECTOR, 'tr.odd, tr.even')
+
+    def remove(self):
```

Comment:
Is this method used anywhere?

Reply:
Not really. I created this object using the same structure as the Workspace since they similar. So I just built this method together which might be used in other test cases